packages/flutter/test/widgets/grid_view_test.dart

```diff
@@ -644,13 +644,50 @@ void main() {
     expect(counters[4], 2);
   });
 
+  testWidgets('GridView does not report visual overflow unnecessarily', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: GridView(
+          gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 3),
+          children: <Widget>[
+            Container(height: 200.0),
```

Comment:
How do you know this isn't overflowing? is it just because 200 < default tester window height?

Reply:
Yes. This is small enough to fit into the viewport.